Added Arabic Language

MahmoudTurkmani opened the pull request.

MahmoudTurkmani committed:
Added Arabic Language

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/padloc/padloc?pullRequest=743) <br/>All committers have signed the CLA.

MahmoudTurkmani committed:
Fixed Filename

The file name was "ar" which turned out to be the country code for Argentina and not Arabic.

Changed it to "sy" which is the closest to the translation that was written.

MahmoudTurkmani committed:
Removed new line at end of file

Just to make the file have the same length as the other translations (no technical reason)

MahmoudTurkmani commented:
@MaKleSoft could you run the tests? I think there was a pending push from my side last time it ran.

MahmoudTurkmani commented:
@MaKleSoft I think it should be OK to merge? The tests ran fine I belive
